backend/akr/src/main/java/fi/oph/akr/service/ClerkTranslatorService.java

```diff
@@ -496,4 +497,57 @@ private Map<LocalDate, ExaminationDate> getLocalDateExaminationDateMap() {
       .stream()
       .collect(Collectors.toMap(ExaminationDate::getDate, Function.identity()));
   }
+
+  public List<ClerkTranslatorDTO> listTranslatorsBySource(final ContactDetailsGroupSource source) {
+    final List<Translator> translators = translatorRepository.findExistingTranslators();
+    final Map<String, PersonalData> personalDatas = onrService.getCachedPersonalDatas();
+    return translators
+      .stream()
+      .map(translator -> {
+        final PersonalData personalData = personalDatas.get(translator.getOnrId());
+        final List<ClerkTranslatorAddressDTO> clerkTranslatorAddressDTO = createTranslatorAddressDTO(
+          personalData,
+          translator
+        );
+        final TranslatorAddressDTO primaryAddress = ContactDetailsUtil.getPrimaryAddress(personalData, translator);
+
+        if (!primaryAddress.source().equals(source)) {
+          return null;
+        }
+
+        if (personalData == null) {
+          LOG.error("Error fetching the translator from onr with oid {}", translator.getOnrId());
+          throw new APIException(APIExceptionType.TRANSLATOR_ONR_ID_NOT_FOUND);
+        }
+
+        return ClerkTranslatorDTO
+          .builder()
+          .id(translator.getId())
+          .version(translator.getVersion())
+          .isIndividualised(personalData.getIndividualised())
+          .hasIndividualisedAddress(personalData.getHasIndividualisedAddress())
+          .firstName(personalData.getFirstName())
+          .lastName(personalData.getLastName())
+          .nickName(personalData.getNickName())
+          .identityNumber(personalData.getIdentityNumber())
+          .email(personalData.getEmail())
+          .phoneNumber(personalData.getPhoneNumber())
+          .address(clerkTranslatorAddressDTO)
+          .extraInformation(translator.getExtraInformation())
+          .isAssuranceGiven(translator.isAssuranceGiven())
+          .authorisations(
+            ClerkTranslatorAuthorisationsDTO
+              .builder()
+              .effective(List.of())
+              .expired(List.of())
+              .expiring(List.of())
+              .expiredDeduplicated(List.of())
+              .formerVir(List.of())
+              .build()
+          )
```

Comment:
Palautetaan tyhjät auktorisoinnit kääntäjälle. Ihan ok varmaankin, kun kerran debug-endpoint kyseessä, mutta oliko tämä tarkoituksella jätetty näin?